src/olympia/activity/serializers.py

```diff
@@ -24,7 +24,8 @@ def __init__(self, *args, **kwargs):
         self.to_highlight = kwargs.get('context', []).get('to_highlight', [])
 
     def get_comments(self, obj):
-        comments = obj.details['comments'] if obj.details else obj.to_string()
+        comments = (obj.details['comments'] if obj.details
+                    else obj.to_string().unescape())
```

Comment:
Since finding the right fix was not trivial, can you add a quick comment explaining why we have to do this ?